platform/podjs/src/browserPod.ts

```diff
@@ -399,12 +398,19 @@ class BrowserNetwork {
 
 interface EndpointInfo {
     url: string;
-    auth: string;
+    auth: null; // TODO: change this to an actual type when it's needed. It's null now on endpoints.json
     allowInsecure: boolean;
 }
 
-function getEndpoint(endpointId: string): EndpointInfo | null {
-    return endpoints[endpointId] || null;
+interface EndpointJSON {
+    polyPediaReport: EndpointInfo;
+    demoTest: EndpointInfo;
+}
```

Comment:
If `endpoints.json` changes format we need to update this type, it's extra maintenance, but adds a line of security as well.